index.js

```diff
@@ -0,0 +1,84 @@
+const recipe={
+    id: 1,
+  title: "Gløgg",
+  picture_url:
+    "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+  ingredients: [
+    { NAME: "Orange zest", AMOUNT: "0.5" },
+    { NAME: "Water", AMOUNT: "200 ml" },
+    { NAME: "Sugar", AMOUNT: "275 g" },
+    { NAME: "Whole cloves", AMOUNT: "5" },
+    { NAME: "Cinnamon sticks", AMOUNT: "2" },
+    { NAME: "Spice", AMOUNT: undefined },
+    { NAME: "Bottle of red wine", AMOUNT: "1" },
+    { NAME: "Raisins", AMOUNT: "100 g" },
+    { NAME: "Slipped Almonds", AMOUNT: "50 g" },
+  ],
+  description: "Mix everything, heat it, and you are good to go!",
+  };
+  
+  function displayRecipe(recipe){
+    const recipeDisplay=document.getElementById("recipeDisplay");
+    let ingredientshtml="";
+    for(let i=0;i<recipe.ingredients.length;i++){
+        const ingredient=recipe.ingredients[i];
+        ingredientshtml +="<li>" + ingredient.NAME + "-" +ingredient.AMOUNT+ "</li>";
```

Comment:
please use Javascript apis to manipulate DOM objects 
https://www.w3schools.com/jsref/dom_obj_li.asp

Reply:
https://frontendinterviewquestions.medium.com/create-ul-and-li-in-javascript-5efa2bbda09b